server/types/relay_entry.go

```diff
@@ -72,3 +73,14 @@ func RelayEntriesToStrings(relays []RelayEntry) []string {
 	}
 	return ret
 }
+
+// Copy returns a deep copy of the relay entry.
+func (r *RelayEntry) Copy() (ret RelayEntry) {
+	ret.PublicKey = r.PublicKey
+	if r.URL != nil {
+		urlCopy := *r.URL
+		ret.URL = &urlCopy
+	}
+	ret.SupportsSSZ = r.SupportsSSZ
```

Comment:
would be a bit cleaner to move it up to line 80, to set all together

Reply:
Sure. I was just doing in the same order as the struct fields.